### PR TITLE
Resolved  Issue - Update logger initialization to use module-specific loggers #140

### PR DESCRIPTION
**Pull request by AkashS0510**
Changed logger = logging.getLogger() to logger = logging.getLogger(__name__) in all files #140

### CODE REVIEW

**Review by samriddhi99 — CHANGES_REQUESTED** [review]
Thanks for the PR! Could you please remove the TODO comments before we merge? Let me know if you need any assistance. 😊

**Comment by AkashS0510** [comment]
Have reomoved the TODO comments

File: src/tirith/core/evaluators/contained_in.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from tirith.utils import sort_collections
 
 # TODO: At least add __name__ as the name for the logger
```
Comment (refeed) [inline src/tirith/core/evaluators/contained_in.py:6]: Hi @AkashS0510, could you also remove the rest `TODO` comments, thanks

Comment (AkashS0510) [reply src/tirith/core/evaluators/contained_in.py]: Apologies for missing these out earlier. I have removed it now. Thank you

Comment (refeed) [inline src/tirith/core/evaluators/contained_in.py:6]: Apologies, could you remove the rest `TODO` comments again, I think that commit got reset previously, thanks

**Review by refeed — CHANGES_REQUESTED** [review]

**Comment by refeed** [comment]
Close and reopening the PR to trigger gh actions

**Review by refeed — APPROVED** [review]
LGTM

**Review by samriddhi99 — APPROVED** [review]

**Pull request merged**